AlpsSDK/ContextManager.swift

```diff
@@ -21,13 +21,17 @@ class ContextManager: NSObject, CLLocationManagerDelegate {
     private weak var delegate: ContextManagerDelegate?
 
     let locationManager = CLLocationManager()
+    private(set) var proximityHandler: ProximityHandler!
```

Comment:
Let's be careful using`!` - this property can be lazy init the same way contextmanager is alpsmanager.

Reply:
Ok will init it lazy.